mocked-api/handlers.ts

```diff
@@ -259,27 +259,28 @@ export const handlers = [
 
     const count = db.processor.count(query);
 
+    const items = db.processor
+      .findMany({
+        take: size,
+        skip: page * size,
+        orderBy: {
+          submitted_at: "desc",
+        },
+        ...query,
+      })
+      .map((item) =>
+        cleanupProcessor(
+          item as unknown as Record<string | number | symbol, unknown>
+        )
+      );
     return res(
       ctx.status(200),
       ctx.delay(apiDelay),
       ctx.json({
         kind: "ProcessorList",
-        items: db.processor
-          .findMany({
-            take: size,
-            skip: page * size,
-            orderBy: {
-              submitted_at: "desc",
-            },
-            ...query,
-          })
-          .map((item) =>
-            cleanupProcessor(
-              item as unknown as Record<string | number | symbol, unknown>
-            )
-          ),
-        page: page,
-        size: size,
+        items,
+        page,
+        size: items.length,
```

Comment:
@vpellegrino Nothing wrong here because we also need this change for the "get processors" handler but you have to do it for "get bridges" too.

Reply:
and errors (if you have a handler for it, IDK).

`Bridges`, `Processors` and `Errors` are the three resources in openapi that support pagination (at the API level).

Reply:
We don't have an handler for Errors yet. I'm not sure we need it if we go with [MGDOBR-669](https://issues.redhat.com/browse/MGDOBR-669) first.